gocourse04/zoo/sector.go

```diff
@@ -0,0 +1,38 @@
+package zoo
+
+import "fmt"
+
+type Sector struct {
+	Subtype string
+	Animals []Animal
+}
+
+func NewSector(subtype string) *Sector {
+	return &Sector{
+		Subtype: subtype,
+		Animals: make([]Animal, 0),
```

Comment:
Ось так трішки коротше:
```suggestion
		Animals: []Animal{},
```